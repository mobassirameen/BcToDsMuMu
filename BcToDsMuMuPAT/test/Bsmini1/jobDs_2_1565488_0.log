000 (1565488.000.000) 04/06 13:40:15 Job submitted from host: <137.138.44.55:9618?addrs=137.138.44.55-9618+[2001-1458-d00-1a--48]-9618&alias=bigbird17.cern.ch&noUDP&sock=schedd_1694_8585_240>
...
040 (1565488.000.000) 04/06 13:43:40 Started transferring input files
	Transferring to host: <128.142.117.10:9618?addrs=128.142.117.10-9618+[2001-1458-301-ae--100-415]-9618&noUDP&sock=6797_d9ed_113483>
...
040 (1565488.000.000) 04/06 13:43:40 Finished transferring input files
...
001 (1565488.000.000) 04/06 13:43:41 Job executing on host: <128.142.117.10:9618?addrs=128.142.117.10-9618+[2001-1458-301-ae--100-415]-9618&noUDP&sock=3553_a415_3>
...
006 (1565488.000.000) 04/06 13:43:46 Image size of job updated: 12804
	0  -  MemoryUsage of job (MB)
	0  -  ResidentSetSize of job (KB)
...
040 (1565488.000.000) 04/06 13:43:47 Started transferring output files
...
040 (1565488.000.000) 04/06 13:43:47 Finished transferring output files
...
005 (1565488.000.000) 04/06 13:43:47 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	21785  -  Run Bytes Sent By Job
	16042  -  Run Bytes Received By Job
	21785  -  Total Bytes Sent By Job
	16042  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :        0        4         4 
	   Disk (KB)            :      535    15000    215729 
	   Memory (MB)          :        0     8000      8000 
...
