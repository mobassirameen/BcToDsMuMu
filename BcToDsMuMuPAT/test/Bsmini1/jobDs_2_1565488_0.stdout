========================
====> SGE  wrapper <====
========================
--> Running SGE digi-reco job wrapper
file:root://se01.indiacms.res.in:1094//cms/store/user/digupta/BctoDsMuMu_MC_MINIAOD_5k/BctoDsMuMu_MC_MINIAOD_5k_10.root
file:BctoDsMuMu_MC_2.root
--> Environment
Tue Apr  6 13:43:42 CEST 2021
b7s06n3322.cern.ch
Linux b7s06n3322.cern.ch 3.10.0-1127.19.1.el7.x86_64 #1 SMP Tue Aug 25 17:23:54 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
Filesystem                     1K-blocks           Used      Available Use% Mounted on
/dev/vda2                       52416492       22921468       29495024  44% /
devtmpfs                         7407276              0        7407276   0% /dev
tmpfs                            7434416             20        7434396   1% /dev/shm
tmpfs                            7434416              0        7434416   0% /sys/fs/cgroup
tmpfs                            7434416           2884        7431532   1% /run
tmpfs                            1486884              0        1486884   0% /run/user/0
AFS                           2147483647              0     2147483647   0% /afs
/dev/mapper/poolvg-poollv      247573692       19254504      215720100   9% /pool
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/cvmfs-config.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/sft.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/atlas.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/na61.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/compass.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/compass-mc.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/na62.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/alice.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/geant4.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/alice-nightlies.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/atlas-online-nightlies.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/atlas-pixel-daq.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/cms-bril.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/cms-ci.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/cms-opendata-conddb.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/compass-condb.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/lhcbdev.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/sft-nightlies.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/alice-ocdb.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/ams.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/atlas-condb.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/atlas-nightlies.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/cms-ib.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/cms.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/ilc.desy.de
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/lhcb-condb.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/lhcb.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/unpacked.cern.ch
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/singularity.opensciencegrid.org
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/oasis.opensciencegrid.org
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/dirac.egi.eu
cvmfs2                          24757248       13403357       11353892  55% /cvmfs/cernvm-prod.cern.ch
experiment                57192306866356 42648807930856 14543498935500  75% /eos/experiment
home-e                     2986232785472  1331771842844  1654460942628  45% /eos/home-e
project-o                  1841406085672   923366007684   918040077988  51% /eos/project-o
cms                       64374846299100 41410339578764 22964506720336  65% /eos/cms
home-v                     2986232785472  1331771842844  1654460942628  45% /eos/home-v
home-h                     2737813750472  1364113233416  1373700517056  50% /eos/home-h
home-y                     2737813750472  1364113233416  1373700517056  50% /eos/home-y
-> which edg-gridftp-ls
/bin/edg-gridftp-ls
-> which globus-url-copy
/bin/globus-url-copy
-> which srmcp
/pool/condor/dir_23417
--> End of env testing
--> Setup CMSSW
/pool/condor/dir_23417
Tue Apr  6 13:43:42 CEST 2021
/pool/condor/dir_23417
Tue Apr  6 13:43:42 CEST 2021
total 60
-rw-------.  1 digupta zh 10843 Apr  6 13:43 x509up_u137973
-rwxr-xr-x.  1 digupta zh  2667 Apr  6 13:43 condor_exec.exe
-rwxr-xr-x.  1 digupta zh  2532 Apr  6 13:43 BcToDsMuMuRootupler.py
drwx------.  3 digupta zh  4096 Apr  6 13:43 var
drwx------.  3 digupta zh  4096 Apr  6 13:43 tmp
-rw-------.  1 digupta zh 13201 Apr  6 13:43 digupta.cc
-rw-r--r--.  1 digupta zh  5337 Apr  6 13:43 _condor_stdout
drwxr-xr-x. 17 digupta zh  4096 Apr  6 13:43 CMSSW_10_2_12
-rw-r--r--.  1 digupta zh   546 Apr  6 13:43 _condor_stderr
/pool/condor/dir_23417
--> Run cmsRun
/pool/condor/dir_23417
Tue Apr  6 13:43:46 CEST 2021
cmsRun step1.py 
condor_exec.exe: line 85: cmsRun: command not found
Tue Apr  6 13:43:46 CEST 2021
/pool/condor/dir_23417
total 64
-rw-------.  1 digupta zh 10843 Apr  6 13:43 x509up_u137973
-rwxr-xr-x.  1 digupta zh  2667 Apr  6 13:43 condor_exec.exe
-rwxr-xr-x.  1 digupta zh  2532 Apr  6 13:43 BcToDsMuMuRootupler.py
drwx------.  3 digupta zh  4096 Apr  6 13:43 var
drwx------.  3 digupta zh  4096 Apr  6 13:43 tmp
-rw-------.  1 digupta zh 13201 Apr  6 13:43 digupta.cc
drwxr-xr-x. 17 digupta zh  4096 Apr  6 13:43 CMSSW_10_2_12
-rw-r--r--.  1 digupta zh  1182 Apr  6 13:43 _condor_stderr
-rw-r--r--.  1 digupta zh    52 Apr  6 13:43 step1.log
-rw-r--r--.  1 digupta zh  6081 Apr  6 13:43 _condor_stdout
total 64
-rw-------.  1 digupta zh 10843 Apr  6 13:43 x509up_u137973
-rwxr-xr-x.  1 digupta zh  2667 Apr  6 13:43 condor_exec.exe
-rwxr-xr-x.  1 digupta zh  2532 Apr  6 13:43 BcToDsMuMuRootupler.py
drwx------.  3 digupta zh  4096 Apr  6 13:43 var
drwx------.  3 digupta zh  4096 Apr  6 13:43 tmp
-rw-------.  1 digupta zh 13201 Apr  6 13:43 digupta.cc
drwxr-xr-x. 17 digupta zh  4096 Apr  6 13:43 CMSSW_10_2_12
-rw-r--r--.  1 digupta zh  1182 Apr  6 13:43 _condor_stderr
-rw-r--r--.  1 digupta zh    52 Apr  6 13:43 step1.log
-rw-r--r--.  1 digupta zh  6667 Apr  6 13:43 _condor_stdout
run: This is the end, my friend
